setup.py

```diff
@@ -62,8 +62,8 @@
 install_requires = [
     'numpy>=1.21.5,<2.1.0',
     'packaging>=21.3.0,<24.2',
-    'torch>=2.5.1,<2.5.2',
-    'triton>=2.1.0',
```

Comment:
we don't need the triton requirement any more right? torch will handle installing the right one

Reply:
The mosaic `pytorch` and `composer` images that we have don't have triton installed, so we need to add them here for imports to work. But for the general OSS user, yes, even if we remove the triton dependency here, torch will install the right one. When the installation is fixed for the next version of torch/composer, we can remove this dependency altogether.

Reply:
ah right, sorry. forgot that detail.